app/src/main/java/com/example/sharingang/NewItemFragment.kt

```diff
@@ -79,6 +79,7 @@ class NewItemFragment : Fragment() {
                 categoryString = resources.getStringArray(R.array.categories)[binding.categorySpinner.selectedItemPosition],
                 latitude = binding.latitude?.toDoubleOrNull() ?: 0.0,
                 longitude = binding.longitude?.toDoubleOrNull() ?: 0.0,
+                sold = false,

```

Comment:
Does the attribute `sold` mean "The item has already been bought by another user" or does it mean "The item is not for sale yet"?

Reply:
`sold = true` means that the item has been bought by another user, and is no longer available.